src/library/basetracktablemodel.cpp

```diff
@@ -489,10 +489,6 @@ QVariant BaseTrackTableModel::rawSiblingValue(
         // FIXME: This should never happen but it does. But why??
         return QVariant();
     }
-    VERIFY_OR_DEBUG_ASSERT(siblingColumn != index.column()) {
```

Comment:
Should this be replaced with a `if` instead of removing the condition check entirely like above? Reading the comment, since it seems the early return was here to prevent infinite recursion. Or is that comment incorrect/out of date?